templates/Dockerfile-rpm.tmpl

```diff
@@ -6,7 +6,8 @@ RUN ( yum install -y yum-plugin-ovl || yum install -y yum-plugin-ovl ) \
     && yum clean all
 
 # install build tools and PostgreSQL development files
-RUN yum groupinstall -y 'Development Tools' \
+RUN yum install -y centos-release-scl-rh epel-release \
```

Comment:
Can you explain this change? I don't understand why we need `epel` and `scl` when we've gotten by without them for so long. What changed in 12?

Reply:
I saw that you introduced them in ff15bc394765d9a92cc054a344414679dfdb8fc1. The change was not made in a template, but in in the centos/oraclelinux pg11 docker files. To still be able to use a template I will try to add steps to install epel and scl in the template